website/src/constants/builtinCodecs.ts

```diff
@@ -0,0 +1,44 @@
+import { AnyCodec, number } from "@ckb-lumos/codec"
+import { blockchain } from "@ckb-lumos/base"
+import { CodecMap } from "@ckb-lumos/molecule"
+import { BIish, BI } from "@ckb-lumos/bi/lib"
+import numeral from "numeral"
+
+function enhanceUnpackBIish(codec: AnyCodec, afterUnpack: (arg: BIish) => string) {
+  return {
+    ...codec,
+    unpack: (packed: Uint8Array) => afterUnpack(codec.unpack(packed)),
+  }
+}
+
+const humanizeBigInteger = (x: BIish) => numeral(BI.from(x).toString()).format("0,0")
```

Comment:
the comment is just an example to show how to humanize, as a JSON, i think we don't need to format it